### ISSUE TITLE
review "Corpses to Coinage" interop

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#105]
Add the required ACTI handling if possible, otherwise document

**Comment by SteveTownsend** [comment]
Test evidence after fix to whitelist this if mod is present
```
J:\GitHub\SmartHarvestSE\src\Data\dataCase.cpp(505): [MESSAGE] {7548} (2020-07-05 09:42:13 AM) Record Corpse To Coinage Corpse Coinage(0xd900aa03) as septims
```
```
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(1057): [VERBOSE] {24288} (2020-07-05 10:07:17 AM) Process REFR 0xff002306 with base object Corpse Coinage/0xd900aa03
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(137): [VERBOSE] {24288} (2020-07-05 10:07:17 AM) 0xd900aa03 0x18(24) [Corpse Coinage] - typename septims
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(395): [VERBOSE] {24288} (2020-07-05 10:07:17 AM) SmartHarvest Corpse Coinage/0xd900aa03 for REFR 0xff002306 at distance 126.01 units
```

**Comment by SteveTownsend** [comment]
fixed in v1.6.1.0
